### PR TITLE
Issue/1109

### PR DESCRIPTION
**Pull request by brian-learningpool**

### CODE REVIEW

**Comment by dancgray** [comment]
+1

File: plugins/content/course/model.schema [hunk 1]
```diff
@@ -42,6 +42,79 @@
...
+          "inputType": {
+            "type": "Boolean",
+            "options": [
+              true,
+              false
+            ]
+          },
+          "validators": [],
+          "title": "Force routing",
+          "help": "When this is set to 'true', the routing will be forced to the apporpriate start page, regardless of the URL"
```
Comment (ryangadams) [inline plugins/content/course/model.schema]: typo here, should be “appropriate”

File: frontend/src/core/scaffold/scaffold.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 define(function(require) {
 
 	var Origin = require('coreJS/app/origin');
+  var _ = require('underscore');
```
Comment (ryangadams) [inline frontend/src/core/scaffold/scaffold.js]: Are the additions on this file indented by one character too many?

**Comment by ryangadams** [comment]
Sorry, a couple of over-picky comments was all the criticism I could come up with.

I approve.

+1

**Pull request merged**